src/main/java/com/palantir/gradle/revapi/RevapiAcceptBreakTask.java

```diff
@@ -0,0 +1,89 @@
+import com.palantir.gradle.revapi.config.GroupNameVersion;
+import com.palantir.gradle.revapi.config.Justification;
+import java.util.Collections;
+import java.util.Optional;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.provider.Property;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+
+public class RevapiAcceptBreakTask extends DefaultTask {
+    private static final String CODE_OPTION = "code";
+    private static final String OLD_OPTION = "old";
+    private static final String NEW_OPTION = "new";
+    private static final String JUSTIFICATION_OPTION = "justification";
+
+    private final Property<ConfigManager> configManager = getProject().getObjects().property(ConfigManager.class);
+    private final Property<String> code = getProject().getObjects().property(String.class);
+    private final Property<String> oldElement = getProject().getObjects().property(String.class);
+    private final Property<String> newElement = getProject().getObjects().property(String.class);
+    private final Property<Justification> justification = getProject().getObjects().property(Justification.class);
+
+    public final Property<ConfigManager> configManager() {
+        return configManager;
+    }
+
+    @Option(option = CODE_OPTION, description = "Revapi change code")
+    public final void setCode(String codeString) {
+        this.code.set(codeString);
+    }
+
+    @Option(option = OLD_OPTION, description = "Old API element")
+    public final void setOldElement(String oldElementString) {
+        this.oldElement.set(oldElementString);
+    }
+
+    @Option(option = NEW_OPTION, description = "New API element")
+    public final void setNewElement(String newElementString) {
+        this.newElement.set(newElementString);
+    }
+
+    @Option(option = JUSTIFICATION_OPTION, description = "Justification for why these breaks are ok")
+    public final void setJustification(String justificationString) {
+        this.justification.set(Justification.fromString(justificationString));
+    }
+
+    @TaskAction
+    public final void addVersionOverride() {
+        ensurePresent(code, CODE_OPTION);
+        ensurePresent(justification, JUSTIFICATION_OPTION);
+
+        configManager.get().modifyConfigFile(revapiConfig ->
+                revapiConfig.addAcceptedBreaks(oldGroupNameVersion(), Collections.singleton(AcceptedBreak.builder()
+                        .code(code.get())
+                        .oldElement(Optional.ofNullable(oldElement.getOrNull()))
+                        .newElement(Optional.ofNullable(oldElement.getOrNull()))
+                        .justification(justification.get())
+                        .build()
+        )));
+    }
+
+    private void ensurePresent(Property<?> prop, String option) {
+        if (!prop.isPresent()) {
+            throw new IllegalArgumentException("Please supply the --" + option + " param to this task");
+        }
+    }
```

Comment:
I think gradle will check this for you if the property wasn't passed

Reply:
It doesn't seem to :(